Discovery/config_dialog.ui

```diff
@@ -246,17 +249,21 @@
    <item row="12" column="1">
     <widget class="QLineEdit" name="editBboxExpr"/>
    </item>
-   <item row="14" column="0" colspan="2">
-    <widget class="QDialogButtonBox" name="buttonBox">
-     <property name="orientation">
-      <enum>Qt::Horizontal</enum>
+   <item row="13" column="1">
+    <widget class="QCheckBox" name="timeCB">
+     <property name="text">
+      <string>Disable display time</string>
      </property>
-     <property name="standardButtons">
-      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
+    </widget>
+   </item>
+   <item row="15" column="0">
+    <widget class="QLabel" name="label_9">
+     <property name="text">
+      <string>Display time</string>
```

Comment:
I just meant to keep them on one line... something like this:

![image](https://cloud.githubusercontent.com/assets/193367/24952551/68041f26-1faa-11e7-9499-6b0db0ce347d.png)